docs/docs/reference/contextual/typeclasses-new.md

```diff
@@ -64,3 +105,6 @@ given readerMonad[Ctx] as Monad[[X] =>> Ctx => X] {
     ctx => x
 }
 ```
+
+
+The conjunction of given instances, extension methods and context bounds allow a concise and natural expression of _typeclasses_
```

Comment:
Stray text?

Reply:
Otherwise LGTM

Reply:
It was meant to be a sort of conclusion, re-using the previous sentence from the docs (since I think it's accurate). 

I'll try to re-write properly